lib/default.json

```diff
@@ -0,0 +1,1390 @@
+{
+  "HeaderImage": "http://assets.tumblr.com/images/default_header/optica_pattern_09.png?_v=abe6f565397f54e880c2b76e6fc2022e",
+  "BackgroundColor": "#FAFAFA",
+  "TitleColor": "#444444",
+  "AccentColor": "#529ECC",
+  "LinkColor": "#529ECC",
+  "TitleFontWeight": "bold",
+  "AvatarShape": "square",
+  "block:ShowAvatar": true,
+  "block:HideAvatar": false,
+  "block:ShowTitle": true,
+  "block:HideTitle": false,
+  "block:ShowDescription": true,
+  "block:HideDescription": false,
+  "block:ShowHeaderImage": true,
+  "block:HideHeaderImage": false,
+  "block:StretchHeaderImage": true,
+  "block:NoStretchHeaderImage": false,
+  "HeaderImage-640": "http://assets.tumblr.com/images/default_header/optica_pattern_09_640.png?_v=abe6f565397f54e880c2b76e6fc2022e",
+  "HeaderImage-1024": "http://assets.tumblr.com/images/default_header/optica_pattern_09_1280.png?_v=abe6f565397f54e880c2b76e6fc2022e",
+  "block:IfHeaderImage": true,
+  "block:IfNotHeaderImage": false,
+  "TitleFont": "'Gibson', sans-serif",
+  "Title": "Untitled",
+  "BlogTitle": "Untitled",
+  "Name": "verycreatorwolf",
```

Comment:
I'm probably going to take `verycreatorwolf` down later, since I don't use tumblr